### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by privatebin-translator**

### CODE REVIEW

File: i18n/de.json
Comment (elrido) [inline i18n/de.json:5]: @rugk & @cryptomilk, I find the German translation of the "ignorance is bliss" slogan awkward and clunky. In other languages other phrases with similar meaning were chosen (see for example #540). Do you mind if we change this in German? Is my suggestion Ok or can you think of a better German phrase?

**Review by elrido — APPROVED** [review]
I'll hold off merging this until other maintainers had a chance to comment on the German slogan change.

File: i18n/de.json
Comment (rugk) [reply i18n/de.json:5]: I agree.

**Pull request merged**
